02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md

```diff
@@ -26,8 +26,10 @@ Included in the [[ITS Theme]].
 ##### How to use the snippet:
 [Readme Guide](https://github.com/SlRvb/Obsidian--ITS-Theme#image-positions)
 
-**Video Explanation:** ["Slides" from the video](https://slrvb.github.io/Site/3_Nebula/ITS-Theme/ITST_Image-Positions/)
-<iframe width="100%" height="250px" src="https://www.youtube.com/embed/VRoBNWvw8sU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+**Video Explanation:**
+["Slides" from the video](https://slrvb.github.io/Site/3_Nebula/ITS-Theme/ITST_Image-Positions/)
```

Comment:
about half of the embedded images on the GitHub Page seem to be dead links?
<img src="https://user-images.githubusercontent.com/73286100/146602198-b9835423-efa8-4634-90cc-333298549ebe.png" alt="image" width=60%>

Reply:
Oh right, I changed some of the titles and didn't update the site 🤦‍♀️. They should load up properly now.